services/util/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Service for utils",
   "main": "src/app.js",
   "devDependencies": {
+    "@types/restify-errors": "~4.3.5",
```

Comment:
I think we don't need the types here, and just realize that they aren't needed in ecosystems either

Reply:
fixed!